src/resources/views/crud/form_content.blade.php

```diff
@@ -7,7 +7,7 @@
 @else
   <div class="card">
     <div class="card-body row">
-      @include('crud::inc.show_fields', ['fields' => $crud->fields()])
+      @include('crud::inc.show_fields', ['fields' => $crud->fieldsWithOverwrittenNamesForHtml($fields)])
```

Comment:
I don't think we should litter the views with `$fieldsWithOverwrittenNamesForHtml()`. I think `$crud->fields()` should return the fields with the name already overwritten. Just like it was before. The difference being WHEN that is done:
- before it was done on `addField()` (the SETTER)
- now it would be done on  `fields()` (the GETTER)

I think 
- that would be much cleaner (no more changes in all the views, since `$crud->fields()` returns the same thing as before;
- less of a breaking change;
- easy to understand;
- easy to read;

Don't you think that would be better @pxpm ?